### DISCUSSION TITLE
What should i do, if i want to set a special normal user?

### DISCUSSION THREAD

**Discussion by Felix1116** [leihs/leihs#1381] [Q&A / Community Support] [answered]
Hi! What should i do, if i want to set a special normal user?
I have 2 Groups and they are all normal user. But a Group is Student und another Group is Employee. The only difference between them is, that Student must be approved after submit orders but Employee can be automatic approved after submit orders.
Where can i set a such role?

**Comment by eins78** [comment, marked as answer]
Sorry, there is no "auto-approval" in Leihs, neither per User, Group, Pool or other settings.
